scripts/pasha_main.coffee

```diff
@@ -260,7 +272,8 @@ module.exports = (robot) ->
             "#{botName} role <role> <name>: assign prio1 roles to people, " +
             "see '#{botName} role help' for details\n" +
             "#{botName} status: display or set prio1 status, " +
-            "see '#{botName} status help' for details"
+            "see '#{botName} status help' for details\n" +
+            "#{botName} contact add|remove <contactRole> <contact>: add and remove emergency contacts\n"
```

Comment:
Can you please list the `contacts` command here as well? Just to make sure it's listed and people finds it.